src/types/User.tsx

```diff
@@ -56,7 +56,7 @@ export enum UserStatus {
   PENDING = 'P',
   ACTIVE = 'A',
   INACTIVE = 'I',
-  BLOCKED = 'B',
+  SUSPENDED = 'B',
```

Comment:
You cannot change the type here.
The ev-server leads it:

```
export enum UserStatus {
  PENDING = 'P',
  ACTIVE = 'A',
  INACTIVE = 'I',
  BLOCKED = 'B',
  LOCKED = 'L',
}
```
Keep the same naming